### ISSUE TITLE
pytorch 1.1.0

### ISSUE THREAD

**Issue by yyfyan** [HRNet/HRNet-Semantic-Segmentation#71]
在1.1.0的分支下，直接使用nn.BatchNorm代替同步BN？是因为1.1.0的BatchNorm层可以自适应多卡？？

**Comment by yyfyan** [comment]
@sunke123

**Comment by sunke123** [comment]
Pytorch-v1.1 provides the conversation between BatchNorm and SyncBatchNorm. https://github.com/HRNet/HRNet-Semantic-Segmentation/blob/bde57b609c0610b38187bc5749f56faa33fe81a4/tools/train.py#L197
